readme.md

```diff
@@ -154,7 +154,21 @@ Defer showing the notification to after the process has exited.
 Type: `string`\
 Default: [See above screenshot](https://github.com/yeoman/update-notifier#update-notifier-)
 
-Message that will be shown when an update is available.
+Message that will be shown when an update is available, support [`pupa`](https://github.com/sindresorhus/pupa) template.
+
+Available placeholders:
+
+- `{name}` - Package name.
+- `{current}` - Current version.
+- `{latest}` - Latest version.
+- `{command}` - Install command.
+
```

Comment:
Should we also expose the update type? https://github.com/yeoman/update-notifier/blob/adf78037fbe4d33f06dd22a36db296902360b590/index.js#L109 We don't use it, but could be useful, I think.

Reply:
I've tried to add this, but seems this is from `options` passed by user, so maybe not useful.